### ISSUE TITLE
[Feature Request] Follow system theme settings

### ISSUE THREAD

**Issue by tgrelka** [ForkIssues/TrackerWin#98]
On Windows, there's now the option to choose between a dark and light theme for OS apps, like settings, explorer etc. Many third-party apps have now the option to automatically adjust their theme based on the chosen theme in the Windows customization setting.

I'd like to see support for this feature added, alongside a new theme switcher - instead of a single button to switch it, I'd rather have a setting where I can choose between the three options:
- Light theme
- Dark theme
- Use Windows theme

![Example theme switcher from Microsoft To-Do](https://user-images.githubusercontent.com/22134144/50573568-b9a0db00-0dd6-11e9-9860-a5fe4be8384e.png)
These options are in German, but translated they read the options above, in the same order.

This is just a minor detail, but definitely a more convenient way, especially if the last option was default -that way, those who prefer a dark theme and have that set up in Windows will automatically experience a dark theme from the beginning, and it may allow adjusting to potential dynamically changing theme features, if ever implemented in Windows itself or by a third-party app.

![Theme switcher button](https://user-images.githubusercontent.com/22134144/50573598-a8a49980-0dd7-11e9-973a-514efd5c5dfc.png)
Keeping this theme switcher button would still be a nice touch, though.

![Theme selection dropdown here](https://user-images.githubusercontent.com/22134144/50573641-439d7380-0dd8-11e9-87bf-5110e5b56717.png)
Instead, add a drop-down/sub menu here with the three options from above.

**Comment by yasinkavakli** [comment]
Can I bump this up? An automatic theme switcher based on Windows settings would be awesome.

I'm using Git-Fork for Windows at work in an enterprise and Git-Fork for Mac at home. Would be awesome to have an automatic theme switcher for Mac and Windows.

Edit: Or at least you could make a reg key that I could add to my task scheduler to change the theme.

**Comment by fleps** [comment]
I would also like to see this, right now Fork dark theme has a blue border around the window that I really dislike, it should follow Windows 10 default (ascent color if enabled).

Is there any plan to add Theme customization? I would love to tweak some stuff, specially line-height / spacing as I find it too cluttered.

Otherwise, great tool, keep up the good work!

**Comment by yasinkavakli** [comment]
Any updates or plans for this @DanPristupov ?
Code introduced this in the last update too, so only Fork is missing here :)

**Comment by DanPristupov** [comment]
Starting 1.51 Fork follows the system theme.

**Comment by MoritzBrueckner** [comment]
@DanPristupov Is it possible to turn that automatic theme following off (I couldn't find it in the preferences)? I like to use the dark theme but everytime I start Fork it is reset to the bright theme.

**Comment by DanPristupov** [comment]
@MoritzBrueckner are you sure the theme resets after each restart? Those things should not be related.

**Comment by MoritzBrueckner** [comment]
Oh you're right, its only after I restart the computer. Consecutive sessions of Fork in one Windows session keep the selected theme. Nonetheless there probably should be a setting to disable that Fork follows the system theme.

**Comment by DanPristupov** [comment]
@MoritzBrueckner we will try to reproduce the behavior after restart.

Fork only follows the system theme when you switch it, otherwise it uses the one you selected in Fork.

**Comment by MoritzBrueckner** [comment]
Thank you :)

If I can help you somehow (testing stuff/builds, or you need more information) don't hesitate to ask. Should I open a new issue for this?

A bit more information:
- I'm using Fork 1.54.1.0 on Windows 10
- This problem exists since the commit which improved theme switching, then there was a hotfix update that looked like it would fix this problem but it wasn't fixed by that (maybe there is some wrong registry/config entry?).
